app/assets/stylesheets/application.css

```diff
@@ -14,9 +14,17 @@
  *= require_self
  */
 
+
 body {
-    background-color: #21b297;
-    padding-top: 10px;
+  background-color: #f7f7e5;
+  color: #007777;
+  font-family: helvetica, verdana, arial, helvetica, sans-serif;
```

Comment:
You have `helvetica` twice. I propose to `font-family: helvetica, verdana, arial, sans-serif;`